index.html

```diff
@@ -9,6 +9,32 @@
 </head>
 
 <body>
+    <div class="container">
+        <header>
+            <nav class="navigation">
+                <div class="logo-wrap">
+                    <a href="index.html"><img src="images/logo 1.svg" alt="logo"></a>
+                </div>
+                <div class="navigation-header">
+                    <a href="">Home</a>
+                    <a href="">About</a>
+                    <a href="">Tech Atack</a>
+                    <a href="">Projects</a>
+                    <a href="">Contact</a>
+                    <a href=""><img src="images/Vector.svg" alt="logo-gthub"></a>
+                    <a href=""><img src="images/ant-design_twitter-circle-filled.svg" alt="logo-twitter"></a>
+                    <a href=""><img src="images/entypo-social_linkedin-with-circle.svg" alt="logo-linkedin"></a>
+                </div>
+            </nav>
+
+            <main class="presentation">
+                <h1 class="text">Hi 👋, <br> My name is <br><span class="highlight">Mikael Elias</span><br>
+                    I build things for web
+                </h1>             
+                <img src="images/IMG_5362-prt.png" alt="foto">
```

Comment:
Como vc colocando as coisas em inglês acho bom a gente alterar o `alt` dessa foto.
```suggestion
                <img src="images/IMG_5362-prt.png" alt="profile picture">
```

Reply:
feito!